palantirnet/search_api_federated_solr
# UMUS-108: hide empty facet options

[UMUS-108](https://palantir.atlassian.net/browse/UMUS-108)

This PR hides a given sidebar facet option if that facet option has 0 results.  Further, if an accordion's content consists solely of empty facet options, then that entire accordion link is also hidden.

### To test (in dev build)
1. Checkout this branch in `src/search_api_federated_solr`
1. CD into the search app `cd react-search`
1. Install dependencies `yarn install`
1. Run the dev task: `yarn start`
1. Observe a new browser window opens and the search app loads
1. Search for 'bones'
1. Expand the "Site Name" accordion and select a site
1. Observe that, upon selection, the other site is hidden.
1. Expand the "Type" accordion and select a type
1. Observe that, upon selection, the other types are hidden.
1. Uncheck the current Site and Type if you haven't already.
1. Expand the "Topic" accordion and select the "Bones and Muscles and Joints (Orthopaedics)" option
1. Observe its affect on the other accordion options
1. Update your search query to 'suicide'
1. Expand the NEW STUFF accordion
1. Confirm you can select both options (because they both apply to the search result)
